openapi/components/schemas/Applications/ApplicationInstanceConfiguration.yaml

```
@@ -0,0 +1,37 @@
+type: object
+required:
+  - settings
+description: |-
+  Application instance configuration.
+
+  Only user can change settings when `configuredBy` is `user`.
+
+  Only the application itself can change settings when `configuredBy` is `application`.
```

nitpick: [clause order](https://product.rebilly.dev/writing-style-guide/grammar-and-mechanics/clause-order/)
```suggestion
  If the `configuredBy` field is set to `user`, only user can change application settings.

  If the `configuredBy` field is set to `application`, only the application itself can change settings.
```